D8.md

```diff
@@ -0,0 +1,79 @@
+NIP-D8
+======
+
+Nostr Key Migration
+-------------------
+
+`draft` `optional`
+
+This NIP defines a simple mechanism which allows users to migrate to a new key and invalidate their current account. It combines signatures, social key recovery, Open Timestamp Attestations, and some trust assumptions to prevent hijacking by an attacker or loss of control through over-reliance on the social graph.
```

Comment:
```suggestion
This NIP defines a simple mechanism which allows users to migrate to a new key and invalidate their current account. It combines signatures, social key recovery, OpenTimestamps Attestations, and some trust assumptions to prevent hijacking by an attacker or loss of control through over-reliance on the social graph.
```